recipes/oneworld/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "oneworld" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 52660254dd1941cebca3f3834df27064fc0f583da87c2358f45ce0ac1d4b46d9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37]
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy
+  run:
```

Comment:
There are version restrictions listed in the `setup.py`, that you should specify under `run`:

```python
install_requires=[ 
          "numpy>=1.18.1",
          "pandas>=1.0.1",
          "matplotlib>=3.2.0",
          "seaborn>=0.10.0",
          "jinja2>=2.11.1",
          "cartopy>=0.17.0"
      ],
```